src/GetPlasterTemplate.ps1

```diff
@@ -36,10 +36,15 @@ function Get-PlasterTemplate {
         [Parameter(Position=0,
                    Mandatory=$true,
                    ParameterSetName="IncludedTemplates",
-                   HelpMessage="Initiates a search for Plaster templates inside of installed modules.")]
+                   HelpMessage="Initiates a search for Plaster templates inside of installed modules. By default, this retrieves from the latest version available for each module.")]
```

Comment:
I'd prefer to keep this help message shorter - considering where it shows up.  But could you add this detail to the command's doc/md file in `docs/en-us/Get-PlasterTemplate.md`?  Same for the new parameter below.  Thanks!

Reply:
Sure I'll clean that up